schemas/list-provisioners-response.yml

```diff
@@ -9,14 +9,36 @@ properties:
       type:       object
       properties:
         provisionerId:
-          title:    "Provisioner ID"
+          title:        "Provisioner ID"
           type:         string
           minLength:    {$const: identifier-min-length}
           maxLength:    {$const: identifier-max-length}
           pattern:      {$const: identifier-pattern}
+        stability:
+          title:        "Stability"
+          description: |
+            This is the stability of the provisioner. Accepted values:
+            'experimental|stable|deprecated'
```

Comment:
descriptions in our stuff can be markdown, so:
```markdown
This is the stability of the provisioner. Accepted values:
 * `experimental`
 * `stable`
 * `deprecated`
```

that way it'll look nicer on the docs site...